python/cudf/cudf/core/groupby/groupby.py

```diff
@@ -38,8 +39,107 @@ def _quantile_75(x):
     return x.quantile(0.75)
 
 
-class GroupBy(Serializable, Reducible, Scannable):
+groupby_doc_template = textwrap.dedent(
```

Comment:
Isn't the point of using `dedent` so that you can write it in a nicely indented way? (The catch is to escape the first newline iirc.)

```python
groupby_doc_template = textwrap.dedent("""\
    Group using a mapper or by a Series of columns.

    A groupby operation involves...

    Parameters
    ----------
    by : ...
    """